Add correlating logs blog (#1769)
opensearch-project/project-website: _authors/deysubho.markdown

---
short_name: deysubho
name: Subhobrata Dey
photo: '/assets/media/authors/deysubho.jpeg'
github: sbcd90 
---

> **Subhobrata Dey** works as a Software Development Engineer in AWS OpenSearch team & primarily works in the area of Security Analytics & Alerting plugins in OpenSearch. He developed the Security Analytics plugin from scratch.

Reviewer: ```suggestion
**Subhobrata Dey** is a Software Development Engineer at AWS focusing on OpenSearch and primarily works on the Security Analytics and Alerting plugins. He developed the Security Analytics plugin from scratch.
```
